lib/librato/rails/version.rb

```diff
@@ -1,5 +1,5 @@
 module Librato
   module Rails
-    VERSION = "0.12.0"
+    VERSION = "0.12.1.beta"
```

Comment:
I generally would recommend against doing the version bump here for reasons mentioned above, usually will do that as part of a one-off commit against master or separate PR if discussion is needed.